src/components/Personal.tsx

```diff
@@ -0,0 +1,75 @@
+import { Fragment, useState } from "react";
+import style from "../components/style//Personal.module.css";
+import { TextField, Typography } from "@mui/material";
+
+let personalData = {};
+
+function Personal(props: any) {
+  const [form, setForm] = useState({});
+
+  function handleForm(e: React.ChangeEvent<HTMLInputElement>, key: string) {
+    setForm({
+      ...form,
+      [key]: e.target.value,
+    });
+  }
+  personalData = {
```

Comment:
Why not use `form` directly? This variable seems redundant.

Reply:
I deleted the variable.And made it work.